File: samples/basicJava/src/main/java/com/namiml/demo/basic/java/AboutActivity.java
package com.namiml.demo.basic.java;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.namiml.demo.basic.java.databinding.ActivityAboutBinding;
import com.namiml.ml.NamiMLManager;

public class AboutActivity extends AppCompatActivity {

    private static final String CORE_CONTENT_LABEL = "About";

    static Intent getIntent(Context context) {
        return new Intent(context, AboutActivity.class);
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityAboutBinding binding = ActivityAboutBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setSupportActionBar(binding.toolbar);
        binding.toolbar.setNavigationIcon(R.drawable.ic_arrow_back);
        binding.toolbar.setNavigationOnClickListener(view -> onBackPressed());
    }

    @Override
    protected void onResume() {
        super.onResume();
        NamiMLManager.enterCoreContent(CORE_CONTENT_LABEL);
    }

    @Override
    protected void onPause() {
        super.onPause();
        NamiMLManager.exitCoreContent(CORE_CONTENT_LABEL);
    }
}

File: samples/basic/src/main/java/com/namiml/demo/basic/AboutActivity.kt
package com.namiml.demo.basic

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.namiml.demo.basic.databinding.ActivityAboutBinding
import com.namiml.ml.NamiMLManager

class AboutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAboutBinding

    companion object {
        private const val CORE_CONTENT_LABEL = "About"
        fun getIntent(context: Context): Intent {
            return Intent(context, AboutActivity::class.java)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAboutBinding.inflate(layoutInflater)
        setContentView(binding.root)
        setSupportActionBar(binding.toolbar)
        binding.toolbar.apply {
            setNavigationIcon(R.drawable.ic_arrow_back_black_24dp)
            setNavigationOnClickListener {
                onBackPressed()
            }
        }
    }

    override fun onResume() {
        super.onResume()
        NamiMLManager.enterCoreContent(CORE_CONTENT_LABEL)
    }

    override fun onPause() {
        super.onPause()
        NamiMLManager.exitCoreContent(CORE_CONTENT_LABEL)
    }
}